### PR TITLE
RDoc-1356 Document ways to add nodes to cluster programatically

### PR DESCRIPTION
**Pull request by iftahbe**

### CODE REVIEW

File: Documentation/4.0/Raven.Documentation.Pages/server/clustering/common-operations.markdown [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Cluster : Common Operations
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/server/clustering/common-operations.markdown]: I'm trying to find something better for name of this article. Options:

- Low-level management
- Low-level cluster API
- Cluster API

Thoughts?

Comment (ayende) [reply Documentation/4.0/Raven.Documentation.Pages/server/clustering/common-operations.markdown]: Managing programatically ?

File: Documentation/4.0/Raven.Documentation.Pages/server/clustering/common-operations.markdown [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+The recommended way to manage a RavenDB cluster is through the Management Studio's [Cluster View](../../studio/server/cluster/cluster-view).
```
Comment (ayende) [inline Documentation/4.0/Raven.Documentation.Pages/server/clustering/common-operations.markdown]:
```suggestion
The easiest way to manage a RavenDB cluster is through the Management Studio's [Cluster View](../../studio/server/cluster/cluster-view).  
```

File: Documentation/4.0/Raven.Documentation.Pages/server/clustering/common-operations.markdown [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+### Powershell
+
+{CODE-BLOCK:powershell}
+Invoke-WebRequest -Method Put -URI "http://<server-url>/admin/cluster/node?url=<new-node-url>&tag=<new-node-tag>&watcher=<is-watcher>&assignedCores=<assigned-cores>"
+{CODE-BLOCK/}
+
+### Linux
+
+{CODE-BLOCK:plain}
+curl -X PUT http://<server-url>/admin/cluster/node?url=<new-node-url>&tag=<new-node-tag>&watcher=<is-watcher>&assignedCores=<assigned-cores>
```
Comment (ayende) [inline Documentation/4.0/Raven.Documentation.Pages/server/clustering/common-operations.markdown]: `curl` is now part of windows, I think we should be able to just use it.

File: Documentation/4.0/Raven.Documentation.Pages/server/clustering/common-operations.markdown (hunk 1 above)
Comment (iftahbe) [reply Documentation/4.0/Raven.Documentation.Pages/server/clustering/common-operations.markdown]: I like Cluster API
And we'll add all the other cluster operations there. (delete, demote, step down, reassign cores, force timeout)

File: Documentation/4.0/Raven.Documentation.Pages/studio/server/cluster/add-node-to-cluster.markdown [hunk 4]
```diff
@@ -45,6 +45,8 @@ Note: This option is only available when the server is in the `Passive` state.
 **4.** Click _Test connection_ to test the connection for the above url entered  
 
 When done, click *Save* to add this server as a node to the cluster  
+
+This operation can also be done programatically. Read more [here](../../../server/clustering/common-operations).
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/studio/server/cluster/add-node-to-cluster.markdown]: This link should be updated to /clustering/cluster-api

Comment (arekpalinski) [reply Documentation/4.0/Raven.Documentation.Pages/studio/server/cluster/add-node-to-cluster.markdown]: I'll fix it after merging

**Pull request merged**